src/main/java/seedu/address/model/person/Meetings.java

```diff
@@ -109,10 +109,37 @@ public String toString() {
         return meetingList.toString();
     }
 
+    /**
+     * Formats the string output of the meetings list to be displayed in the DetailPanel.
+     * @return String output of meetings list
+     */
+    public String toDetailPanelString() {
```

Comment:
Could the naming of this method be clearer? What to detail panel???

Reply:
i think this is as clear as it can get though. Because it's basically just another toString method that is meant for the DetailPanel